testdata/src/a/a.go

```diff
@@ -49,3 +49,11 @@ func ExampleKlog() {
 	logger := klog.NewKlogr()
 	logger.Info("message", "key1") // want `odd number of arguments passed as key-value pairs for logging`
 }
+
+func ExampleZap() {
+	logger, _ := zap.NewProduction()
+	defer logger.Sync()
+	sugar := logger.Sugar()
+	sugar.Errorw("failed to fetch URL", "url", "https://example.com", "attempt", 3, "backoff", "1s")
+	sugar.With("foo", "bar").Infow("this is a info message", "key1", "value1", "key2", "value2")
```

Comment:
You should add a "fail" case which tests whether the linter is actually working (`// want: xxxxxx`)